File: 2396-strictly-palindromic-number/2396-strictly-palindromic-number.c
bool isStrictlyPalindromic(int n) {
    int n_tmp = n;
    int temp;
    for (int b = 2; b < n - 1; b++) {
        temp = 0;
        n_tmp = n;
        while (n_tmp) {
            temp = temp * b + n_tmp % b;
            n_tmp /= b;
        }
        if (temp != n) return 0;
    }
    return 1;    
}

File: 2396-strictly-palindromic-number/2396-strictly-palindromic-number.py
class Solution:
    def isStrictlyPalindromic(self, n: int) -> bool:
        n_tmp = n
        
        for b in range(2, n - 1):
            temp = 0
            while n_tmp:
                temp = temp * b + n_tmp % b
                n_tmp /= b
            if temp != n: return False
            
        return True
                
        
        